synapse/storage/database.py

```diff
@@ -446,6 +446,10 @@ def __init__(
                 self._check_safe_to_upsert,
             )
 
+    def name(self) -> str:
+        "Return the name of this database"
+        return self._database_config.name
```

Comment:
This could be a `@property`

Reply:
Yeah, though I'm not a huge fan of using properties for the sake of it :shrug: